straxen/plugins/events/_event_s1_positions_base.py

```diff
@@ -20,7 +20,6 @@ class EventS1PositionBase(strax.Plugin):
 
     algorithm: Optional[str] = None
     compressor = "zstd"
-    parallel = True
```

Comment:
The `parallel = True` is not necessary because we are deprecating multi-processes mode.